pkg/operator/ceph/object/spec.go

```diff
@@ -104,6 +105,34 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) v1.PodTemplateSpec
 	return podTemplateSpec
 }
 
+func (c *clusterConfig) setPodPlacement(pod *v1.PodSpec, p rook.Placement) {
+	p.ApplyToPodSpec(pod)
+
```

Comment:
Just like in the mon implementation, if a node selector is enforced in the CR we should return here and not do anything.

Reply:
I don't think that nodeSelector is available in the CR.
For the mon it is usefull because the nodeSelector is enforced by the operator and should be taken in account when applying pod anti affinity.

Reply:
Oops, I was thinking about the placement. We don't need this.